source/core/src/main/com/csse3200/game/components/npc/attack/RangeAttackComponent.java

```diff
@@ -110,7 +118,8 @@ public void performAttack() {
         entity.getEvents().trigger("attack");
         String baseName = this.getEntity().getComponent(NameComponent.class).getName();
         // Uncomment this for testing change projectile animation
-//        setAnimationID(0);
+        // Primarily for Kitsune
+        setAnimationID((int) (Math.random() + 1));
```

Comment:
use ServiceLoactor.getRandom() instead of Math.random

Reply:
Was a useless random and has been removed